src/main/resources/util/thresholds.jelly

```diff
@@ -130,14 +130,14 @@
             </tbody>
           </table>
         </f:entry>
-        <f:entry title="${%Use delta for new warnings}" description="${%description.useDeltaValues}">
```

Comment:
True. I'll simply drop this commit as it's too involving for me to update all translations.